manifest/schema.go

```diff
@@ -40,13 +40,18 @@ type Mount struct {
 	// The proxy destination used when handling requests.
 	// Mutually exclusive with Content option.
 	Proxy string
-	// If PathIsPrefix is true and ProxyAppendSuffix is true, the suffix to Path Prefix will also be appended to Proxy.
+	// If PathIsPrefix is true and AppendSuffix is true, the suffix to Path Prefix will also be appended to Proxy Or LocalDir.
 	// Otherwise, it will be many to one proxy.
-	ProxyAppendSuffix bool `yaml:"proxyAppendSuffix"`
+	AppendSuffix bool `yaml:"appendSuffix"`
 
 	// Provides content template (passed through template/text) to serve.
 	// Mutually exclusive with Proxy option.
 	Content string
+
+	// Provides a path on the host to find the files.
+	// So that LocalDir: /tftpboot path: /subdir and client requests: /subdir/file.x the path on the host
+	// becomes /tfptboot/subdir/file.x
```

Comment:
In this case the path would become `/tftpboot/file.x` (probably forgot to update after the latest change).

Reply:
```suggestion
	// becomes /tfptboot/file.x
```